src/Controller/PageAdminController.php

```diff
@@ -119,7 +119,7 @@ public function treeAction(Request $request): Response
         $theme = $this->admin->getFilterTheme();
         $this->setFormTheme($formView, $theme);
 
-        return $this->renderWithExtraParams($this->admin->getTemplateRegistry()->getTemplate('tree'), [
+        return $this->renderWithExtraParams('@SonataPage/PageAdmin/tree.html.twig', [
```

Comment:
Kinda weird. Doing this won't allow people to override the template by using the template registry.

Reply:
hmmmm But can't you overwrite like this

`templates/bundles/SonataPage/PageAdmin/tree.html.twig` ?

Reply:
What if you want to override the template for only one admin ?
See https://github.com/sonata-project/SonataPageBundle/issues/1639#issuecomment-1300000091, there is something to look at in the configuration, about decorated services.

Reply:
I don't see the benefit of this change. Even with the old line of code, if you overwrite the template adding a new one with the same name on the `bundles/SonataPage/PageAdmin` it should work.

Reply:
Hey @jordisala1991, it was just a try to fix the page admin when I'm decorating the PageAdmin service!

it's the exception: https://github.com/sonata-project/SonataPageBundle/issues/1639#issuecomment-1299912766